admin/load.php

```diff
@@ -336,3 +336,88 @@ function perflab_get_module_data( $module_file ) {
 
 	return $module_data;
 }
+
+/**
+ * Initialise Admin Pointer
+ *
+ * Handles the bootstrapping of the admin pointer.
+ * Mainly jQuery code that is self-initialising.
+ *
+ * @since 1.0.0
+ * @return void
+ */
+function perflab_admin_pointer() {
+
+	wp_enqueue_style( 'wp-pointer' );
+	wp_enqueue_script( 'wp-pointer', false, array( 'jquery' ) );
```

Comment:
We must be careful here not to unconditionally enqueue anything. If the pointer is dismissed, we need to bail early. Based on my other comments, I'd suggest the following:
```suggestion
function perflab_admin_pointer() {
	$pointer_id = 'perflab-intro-pointer';
	$dismissed  = explode( ',', (string) get_user_meta( get_current_user_id(), 'dismissed_wp_pointers', true ) );
	if ( in_array( $pointer_id, $dismissed, true ) ) {
		return;
	}
	
	wp_enqueue_style( 'wp-pointer' );
	wp_enqueue_script( 'wp-pointer', false, array( 'jquery' ) );
```

Reply:
I've updated this, thanks @felixarntz !